scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/fgboost/common/TreeUtils.scala

```diff
@@ -14,13 +14,22 @@
  * limitations under the License.
  */
 
-package com.intel.analytics.bigdl.ppml.vfl.fgboost
+package com.intel.analytics.bigdl.ppml.fgboost.common
+
+
 
 import com.intel.analytics.bigdl.dllib.tensor.Tensor
 
+
 object TreeUtils {
 
   // TODO sort dataset by feature, sort index of sorted result
+  /**
+   * For each feature, sort from smallest to largest value of the inputData, and return an Array
+   * For N * M (batchSize * featureNum) the result should be (featureNum * batchSize)
```

Comment:
`batchSize * featureNum` or `featureNum * batchSize`?

Reply:
The result is `featureNum * batchSize`, the shape is transposed